libs/ui/atoms/src/lib/Icon/Icon.tsx

```diff
@@ -80,126 +80,139 @@ enum IconType {
 }
 
 interface IIconProps {
+  size?: number | undefined;
```

Comment:
To every place with 'undefined': could we remove it? Do we need that? Isn't a question mark enough, when we just don't pass the prop?

Reply:
We get any error when we trying to build it so we need it.

Reply:
I looked into some props from native base lib and people from this project has the same approach. I based on better programmers from me :P